docs/manage-application.md

```diff
@@ -0,0 +1,78 @@
+---
+id: manage-application
+title: Manage a ReSolve Application
+---
+
+Use the [resolve-module-admin](https://github.com/reimagined/resolve/tree/master/packages/modules/resolve-module-admin#readme) CLI tool to manage a reSolve application's sagas and read models. It includes the following functionality:
+
+- List application read models and sagas.
+- Pause and resume updates for read models and sagas.
+- Reset read model and saga persistent state.
+- Manage saga properties.
+
+## Installation
+
+Use the steps below to add the **resolve-module-admin** tool to a reSolve application.
+
+1. Install the [resolve-module-admin](https://github.com/reimagined/resolve/tree/master/packages/modules/resolve-module-admin#readme) NPM package:
+
+   ```bash
+   yarn add resolve-module-admin
+   ```
+
+2. Register a script that runs the **resolve-module-admin** tool in the application's **package.json** file:
+
+   ```js
+   "scripts": {
+     ...
+     "resolve-module-admin": "resolve-module-admin",
```

Comment:
Maybe it's better to change the script to 'admin' or 'manage' or something similar for convenience.
Command will look better in this case, for example: 
yarn manage read-models reset ShoppingList